src/main/java/org/terasology/core/world/generator/facetProviders/SpawnPlateauProvider.java

```diff
@@ -55,9 +55,9 @@ public SpawnPlateauProvider(Vector2ic center) {
     @Override
     public void process(GeneratingRegion region) {
         BlockRegion reg = region.getRegion();
-        Rect2i rc = Rect2i.createFromMinAndMax(reg.minX(), reg.minZ(), reg.maxX(), reg.maxZ());
+        Rectanglei rc = new Rectanglei(reg.minX(), reg.minZ(), reg.maxX(), reg.maxZ());
```

Comment:
why does this need to be cast to Rectanglei? when you can use a BlockArea?

Reply:
See discussion on https://github.com/Terasology/CoreWorlds/pull/25#discussion_r567191415